src/util/formDevices.tsx

```diff
@@ -45,8 +52,23 @@ export const formDeviceToPayload = (devices: FormDevice[]) => {
 export const parseDevices = (devices: LxdDevices): FormDevice[] => {
   return Object.keys(devices).map((key) => {
     const item = devices[key];
+
+    const isCustomNetwork =
+      item.type === "nic" &&
+      Object.keys(item).filter(
+        (key) => !["type", "name", "network"].includes(key)
+      ).length > 0;
```

Comment:
I think this can be simplified to:
```suggestion
    const isCustomNetwork =
      item.type === "nic" &&
      Object.keys(item).some(
        (key) => !["type", "name", "network"].includes(key)
      );
```